<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPanel</class>
 <widget class="QWidget" name="EditPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progress Panel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="batch_change_btn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>批量分派人员</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="client_pass">
       <property name="text">
        <string>客户通过</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="client_no_pass">
       <property name="text">
        <string>暂未通过</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="project_list">
      <property name="minimumSize">
       <size>
        <width>85</width>
        <height>0</height>
       </size>
      </property>
     </widget>
     <widget class="QTableWidget" name="detail_table">
      <property name="minimumSize">
       <size>
        <width>1250</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
